exporter_test.go

```diff
@@ -655,6 +655,23 @@ type = "Apache-2.0"
 				nonExistingOriginalImage.Cleanup()
 			})
 
+			it("create a slice layer on the Run image", func() {
+				h.AssertNil(t, ioutil.WriteFile(filepath.Join(appDir, "static", "resources", "assets", "config.txt"), []byte("app-config"), os.ModePerm))
+				h.AssertNil(t, ioutil.WriteFile(filepath.Join(appDir, "static", "resources", "assets", "image.jpeg"), []byte("app-image"), os.ModePerm))
+
+				h.AssertNil(t, exporter.Export(layersDir, appDir, fakeAppImage, runImageRef, metadata.LayersMetadata{}, additionalNames, launcherConfig, stack))
+
+				sliceLayerPath := fakeAppImage.AppLayerPath()
```

Comment:
Need to think this through.  With the introduction of additional Slice layers, the image layer paths are now getting staggered by the number of slice layers that get saved to the image.  For this particular test, a single `slice` layer was introduced, causing the `AppLayerPath` (hardcoded to index 0) to now point to the `slice` layer and the `ConfigLayerPath` (hardcoded to index 1) now points to the app layer.   Will probably need to make changes to the `imageUtil` lib.